terraform/modules/gro_ingestion_service/lambdas/package.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "gro-ingestion-lambda",
+  "version": "0.0.1",
+  "description": "GRO Ingestion Lambda",
+  "author": "gdx-dev-team@digital.cabinet-office.gov.uk",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "tsc"
+  },
+  "devDependencies": {
+    "@types/aws-lambda": "^8.10.114",
```

I have a preference for pinning to exact versions, e.g.

```suggestion
    "@types/aws-lambda": "8.10.114",
```

there's an npmrc config setting for this I'll try and find...

---

found it